dist/amd/typeahead/aubs-typeahead.js

```diff
@@ -201,6 +201,7 @@ define(['exports', 'aurelia-framework', '../utils/bootstrap-options'], function
             }
 
             this.dropdown.classList.add(this.showClass);
+            this.menu.classList.add(this.showClass);
```

Comment:
This is going to break in Bootstrap 3 mode. Because `menu` will be `null` we need to check in what mode we are to add and remove the class from the right element